expo/gbe/views/create_volunteer_view.py

```diff
@@ -70,24 +115,38 @@ def CreateVolunteerView(request):
                 messages.success(request, user_message[0].description)
             return HttpResponseRedirect(reverse('home', urlconf='gbe.urls'))
         else:
+            formset += [form]
+            if not like_one_thing:
+                user_message = UserMessage.objects.get_or_create(
+                    view='CreateVolunteerView',
+                    code="NO_INTERESTS_SUBMITTED",
+                    defaults={
+                        'summary': "Volunteer Has No Interests",
```

Comment:
We'll have to figure out a reasonable way to handle these literal strings, but not a blocker at the moment.